### PR TITLE
Make npm, bower and composer install optional

### PR DESCRIPTION
**Pull request by KristiyanHristov**
closes #42 

- Makes apt packages configurable
- Makes npm packages configurable

### CODE REVIEW

**Comment by KristiyanHristov** [comment]
README should be updated

File: ansible/services.yml [hunk 1]
```diff
@@ -123,16 +135,10 @@
 
     - name: Install nodejs, git, make
```
Comment (venimus) [inline ansible/services.yml]: This will be no more relevant

File: ansible/services.yml [hunk 2]
```diff
@@ -14,6 +14,18 @@
 
     nodejs_version: '6.x'
 
+    has_npm: true
+
+    apt_packages:
+      - nodejs
```
Comment (venimus) [inline ansible/services.yml]: I don't think nodejs should be here

File: ansible/services.yml [hunk 3]
```diff
@@ -14,6 +14,18 @@
 
     nodejs_version: '6.x'
 
+    has_npm: true
```
Comment (venimus) [inline ansible/services.yml]: There should be dedicated nodejs + npm install

File: ansible/services.yml [hunk 4]
```diff
@@ -123,16 +135,10 @@
...
-        pkg:
-          - nodejs
-          - git
-          - make
-          - unzip
-          - npm
+        pkg: "{{ apt_packages }}"
         state: present
 
     - name: Install bower & yarn
```
Comment (venimus) [inline ansible/services.yml]: install npm packages

File: ansible/services.yml [hunk 5]
```diff
@@ -14,6 +14,17 @@
 
     nodejs_version: '6.x'
 
+    apt_packages:
+      pkg:
+        - nodejs
+        - git
+        - make
+        - unzip
```
Comment (KristiyanHristov) [inline ansible/services.yml:22]: I don't pass pass an array directly to handle this case: (a downgraded package) 

```
apt_packages:
      - { name: nodejs, state: present }
      - { name: git, state: present }
      - { name: make, state: present }
      - { name: unzip, state: present }
      - { name: php-ast=0.1.6-1, dpkg_options: "force-confdef,force-confold,force-downgrade", state: present }
```

File: README.md [hunk 6]
```diff
@@ -123,6 +123,13 @@ pagrant:
     php_fpm_version: '7.2' # Define the PHP version. Default is '7.1'.
     nodejs_version: '8.x' # Define the NodeJS major version. Default is '6.x'.
     nginx_sites_default_root: /app/public # Absolute path of the public dir. Default is '/app/web'.
+    apt_packages:
+      pkg:
+        - npm
```
Comment (venimus) [inline README.md]: put the defaults here (and mention it as comment)
This is designed to be copy-pasted so don't break that

Comment (venimus) [reply README.md]: In fact better make different section for it

**Review by venimus — CHANGES_REQUESTED** [review]

**Review by venimus — APPROVED** [review]

**Review by dincho — APPROVED** [review]

**Pull request merged**